### PR TITLE
Jormun: Fix distributed kraken

### PR DESCRIPTION
**Pull request by AurelienLP**
After streetnetwork_backends were added in database, distributed with kraken was broken because we didn't call the right kraken for the requested coverage.

To fix that, we now send the instance to the service to use the right socket hence the right kraken.
(direct_path_with_fp and get_street_network_routing_matrix)

QA test : 
1) Request a journey with distributed on any coverage using kraken with distributed mode.
2) Request a journey with distributed on an OTHER coverage using kraken with distributed mode.
3) You have an error there. You shouldn't.

### CODE REVIEW

**Comment by AurelienLP** [comment]
Jenkins, test this please.

**Comment by AurelienLP** [comment]
Jenkins, test this please.

**Comment by AurelienLP** [comment]
Jenkins, test this please.

**Review by nonifier — COMMENTED** [review]
A unit test that shows the behaviour that you're trying to fix is most welcome :)

File: source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py [hunk 1]
```diff
@@ -106,13 +106,15 @@ def _get_duration(self, resp, place):
     @new_relic.distributedEvent("routing_matrix", "street_network")
     def _get_street_network_routing_matrix(self, origins, destinations):
         try:
-            return self._instance.get_street_network_routing_matrix(
+            if not self._streetnetwork_service:
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py]: this case can happen? if yes  we should raise an exception as it look more like an error don't you think?

**Review by kinnou02 — COMMENTED** [review]
A lot of boring changes :D

File: source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py (hunk 1 above)
Comment (AurelienLP) [reply source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py]: I should throw a `TechnicalError` in `StreetNetworkBackendManager::get_street_network_db` if it is `None` in that case. Hence this line will be useless. Thanks!

**Review by pbougue — COMMENTED** [review]
Could you also describe a QA-test currently failing on one of our platforms (`dev` preferably) to be able to test it once merged ?

This PR must be rebased :disappointed:

This looks OK, however I'd feel safer if @kinnou02 or @xlqian approves

**Review by pbougue — APPROVED** [review]

**Comment by AurelienLP** [comment]
> Could you also describe a QA-test currently failing on one of our platforms (dev preferably) to be able to test it once merged ?

I am actually trying to implement a integration test but I am not able to reproduce the issue yet.

**Review by kinnou02 — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Pull request merged**